city_scrapers/spiders/lawa.py

```
@@ -0,0 +1,179 @@
+            )
+
+            meeting["classification"] = self._parse_classification(
+                item, meeting["title"]
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title = item.xpath("./td[@headers='Name']/text()")
+        size = len(title)
+        if size < 1:
+            return ""
+        return title[size - 1].extract().strip()
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item, title):
+        """Parse or generate classification from allowed options."""
+        if "committee" in title.lower():
+            return COMMITTEE
+        return BOARD
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object.
+        Default start times:
+        regular board meeting - 10am
+        security committee - 12pm
+        audit committee - 12:30pm"""
+        title = item.xpath("./td[@headers='Name']/@id")
+        date_tag = ""
+        if len(title) >= 1:
+            date_tag = title[0].extract()
+
+        path = "./td[@headers='Date " + date_tag + "']/text()"
+        date = item.xpath(path)
+        if len(date) < 1:
+            return datetime(1, 1, 1, 0, 0)
+
+        date_text = date[0].extract().strip()
+
+        if "special" in date_tag.lower():
+            pass
+        elif "audit" in date_tag.lower():
+            date_text = date_text + " 12:30pm"
+        elif "security" in date_tag.lower():
+            date_text = date_text + " 12pm"
+        elif "regular" in date_tag.lower():
+            date_text = date_text + " 10am"
+
+        try:
+            return dateparse(date_text, fuzzy=True, ignoretz=True)
+        except (ParserError):
+            return datetime(1, 1, 1, 0, 0)
```

We might have to change this in the future depending on our research into why some of the scrapers are failing silently